pkg/detectors/auth0oauth/auth0oauth.go

```diff
@@ -94,11 +84,7 @@ func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (result
 						}
 						body := string(bodyBytes)
 
-						// if client_id and client_secret is valid -> 403 {"error":"invalid_grant","error_description":"Invalid authorization code"}
-						// if invalid -> 401 {"error":"access_denied","error_description":"Unauthorized"}
-						// ingenious!
-
-						if !strings.Contains(body, "access_denied") {
+						if !bytes.Contains(bodyBytes, []byte("access_denied")) {
```

Comment:
Suggestion: https://github.com/trufflesecurity/trufflehog/blob/main/pkg/common/utils.go#L42-L53 would be more efficient